### PR TITLE
[MIG]account_bank_statement_import_paypal: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
Based on https://github.com/OCA/bank-statement-import/pull/144

cc @Tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@sebastienbeau maybe you can review this. It's for the OCA instance.

File: account_bank_statement_import_paypal/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+In the menu Accounting > Configuration > Accounting > Bank Accounts,
+make sure that you have your Paypal bank account with the following parameters:
+* Bank Account Type: Normal Bank Account
+* Account Number: the email address associated with your Paypal account
+* Account Journal: the journal associated to your Paypal account
+
+TIPS
+----
+For now only French and English report are supported.
```
Comment (chienandalu) [inline account_bank_statement_import_paypal/readme/CONFIGURE.rst:9]: @cubells Could we have them in spanish as well?

Comment (pedrobaeza) [reply account_bank_statement_import_paypal/readme/CONFIGURE.rst:9]: That's out of the scope of the module right now. This is for the OCA instance

Comment (chienandalu) [reply account_bank_statement_import_paypal/readme/CONFIGURE.rst:9]: :+1:

**Comment by pedrobaeza** [comment]
@elicoidal @sbidoul maybe you can review this that it's for the OCA instance.

**Pull request merged**